boards/st/stm32mp257f_ev1/stm32mp257f_ev1_stm32mp257fxx_m33.dts

```diff
@@ -66,6 +66,14 @@
 	status = "okay";
 };
 
+
+&spi3 {
```

Comment:
Why enable this and only this SPI instance? Is something connected to there on the board and not yet supported in Zephyr?